portality/settings.py

```diff
@@ -1,5 +1,7 @@
 import os
```

Comment:
we'll need a version bump too

Reply:
Can we find a way to automate this so we don't have to remember to do it at rollout?  I'll open an issue, actually ...

Reply:
Build system? Otherwise it's checked in so requires a commit. Or we could cache-bust every deployment, make it part of the deployment process to write a new value to a file that gets read in on app restart.